content/webapp/services/prismic/api.ts

```diff
@@ -0,0 +1,40 @@
+import * as prismic from 'prismic-client-beta';
+import { PrismicDocument } from '@prismicio/types';
+import { GetServerSidePropsContext } from 'next';
+import fetch from 'node-fetch';
+
+type Req = GetServerSidePropsContext['req'];
+
+const routes = [
+  {
+    type: 'seasons',
+    path: '/seasons/:uid',
+  },
+];
+const endpoint = prismic.getEndpoint('wellcomecollection');
+const client = prismic.createClient(endpoint, { routes, fetch });
+
+// We enforce have a req object as we should never be using the API on the client.
+// To fetch stuff via the client, use the ./api paths.
+const initialised = false;
+export function api(req: Req): prismic.Client {
+  if (!req) {
+    throw Error('Give me a req');
+  }
+  if (initialised) {
+    throw Error(
+      'Prismic has already been initialised, please use the previously initialised API reference'
+    );
+  }
+
+  return client;
+}
+
+export async function getDocument<Doc extends PrismicDocument>(
```

Comment:
what's the purpose of this function? it doesn't seem to do anything that `client.getById<Doc>(id)` doesn't

Reply:
Good point, I have kept the `getSeason` method though with a type guardian.